examples/hlb_cifar10.py

```diff
@@ -127,7 +129,8 @@ def _cov(X):
     def _patches(data, patch_size=(kernel_size,kernel_size)):
       h, w = patch_size
       c = data.shape[1]
-      return np.lib.stride_tricks.sliding_window_view(data, window_shape=(h,w), axis=(2,3)).transpose((0,3,2,1,4,5)).reshape((-1,c,h,w))
+      axis: SupportsIndex = (2, 3) # type: ignore
```

Comment:
Hmm, why? Does the array [2,3] does work?

Reply:
Nope. It's very weird. Int works, but not tuple or list